### PR TITLE
Typescript for Homepage

### PR DESCRIPTION
**Pull request by hedgehog1833**
My first TypeScript page 🥳 
I started with the row for recently followed artists.

### CODE REVIEW

File: webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    private attachCard(divElement: HTMLDivElement, rowElement: HTMLDivElement) {
+        rowElement.insertAdjacentElement("beforeend", divElement);
+    }
+
+    private createFollowedSinceString(followedSince: Date): string {
+        const now = Date.now();
+        const then = new Date(followedSince).valueOf();
+        const followedForDays = Math.round((now - then) / 1000 / 60 / 60 / 24);
```
Comment (DanielW1987) [inline webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts:50]: Isn't there some cool JS function for?

File: webapp/src/main/resources/templates/frontend/home.html [hunk 2]
```diff
@@ -6,261 +6,28 @@
...
-                        <p class="h5 card-title">Alcest</p>
-                    </div>
-                    <div class="card-footer text-muted">
-                        41 days ago
-                    </div>
-                </div>
-            </div>
+        <div id="home-container" class="mb-3">
+            <h2 class="h5 mt-4 mb-2">Recently followed artists</h2>
+            <div id="recently-followed-row" class="row"></div>
```
Comment (DanielW1987) [inline webapp/src/main/resources/templates/frontend/home.html:30]: And what does it look like if I don't follow artists at all? Do I only see a headline without content? That shouldn't be.

File: webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts (hunk 1 above)
Comment (NilsDammenhayn) [reply webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts:50]: Couldn't find one after a short google search...

File: webapp/src/main/resources/templates/frontend/home.html (hunk 2 above)
Comment (NilsDammenhayn) [reply webapp/src/main/resources/templates/frontend/home.html:30]: Well I could of course don't display anything if it's empty but wasn't the plan to show a placeholder if it is empty anyway?

Comment (DanielW1987) [reply webapp/src/main/resources/templates/frontend/home.html:30]: In trello It is specified that we only display the placeholder if there is at least 1 "real" entry. You can fix it with the next PR.

File: webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts (hunk 1 above)
Comment (DanielW1987) [reply webapp/src/main/resources/static/ts/src/service/homepage-render-service.ts:50]: Maybe you can introduce a date util class with the next PR.

**Comment by hedgehog1833** [comment]
[[Detector] Render start page cards with TypeScript](https://trello.com/c/D9Na8oxO/207-detector-render-start-page-cards-with-typescript)

**Pull request merged**